static/js/blame.js

```diff
@@ -156,22 +156,42 @@ var BlameStripHoverHandler = new (class BlameStripHoverHandler {
     // element will null this out as a means of letting the user get rid of the
     // popup if they didn't actually want to see the pop-up.
     this.mouseElement = null;
+    // Set to true if the user clicks on the blame strip. This will keep the
+    // keep the popup visible until the user clicks elsewhere.
+    this.keepVisible = false;
 
     for (let element of document.querySelectorAll(".blame-strip")) {
       element.addEventListener("mouseenter", this);
       element.addEventListener("mouseleave", this);
-      element.addEventListener("click", this);
     }
 
     BlamePopup.popup.addEventListener("mouseenter", this);
     BlamePopup.popup.addEventListener("mouseleave", this);
+    window.addEventListener("click", this, {capture: true});
+    document.getElementById("scrolling").addEventListener("scroll", this, {capture: true});
   }
 
   handleEvent(event) {
     // Suppress the blame hover popup if the context menu is visible.
     if (ContextMenu.active) {
       return;
     }
+
+    if (this.keepVisible) {
+      if (event.type == "mouseenter" || event.type == "mouseleave") {
+        // Ignore mouseenter/mouseleave events if keepVisible is true
+        return;
+      }
+    }
+
+    let clickedOutsideBlameStrip =
+        event.type == "click" &&
+        event.originalTarget.closest(".blame-strip") == null;
```

Comment:
`.originalTarget` is Firefox-specific, probably `.target` works.

Reply:
Also probably `!...closest()` instead of `== null`?

Reply:
And actually, you need to make it `!....matches(".blame-strip")`. `.blame-strip` elements are empty anyways, so it's cheaper, and the code relies on the `blameElement` being the `.blame-strip` element being the target element.